lib/redis_client.py

```diff
@@ -0,0 +1,40 @@
+import os
+import redis
+import datetime
+from nypl_py_utils.functions.log_helper import create_log
+
+
+class RedisClient:
+    def __init__(self):
+        self.base_client = self._create_base_client()
+        self.logger = create_log('redis_client')
+
+    def key_for_hold(self, entry):
+        return 'item-status-listener-processed-hold-{}'\
+            .format(entry['hold_id'])
+
+    def get_hold_processed(self, entry):
+        key = self.key_for_hold(entry)
+        result = self.base_client.get(key)
+        if result is not None:
+            self.logger.debug(
+                '  Skip hold {}, already processed on {}'
+                .format(entry['hold_id'], result)
+            )
+        print('Result: {}'.format(result))
+        return type(result) == str
```

Comment:
Is this just to check that the result isn't None or is there a possibility that it could be an int or something?